example/ndpiReader.c

```diff
@@ -1401,11 +1403,14 @@ static void printFlow(u_int32_t id, struct ndpi_flow_info *flow, u_int16_t threa
     if(flow->tunnel_type != ndpi_no_tunnel)
       fprintf(out, "%s:", ndpi_tunnel2str(flow->tunnel_type));
 
-    fprintf(out, "%s/%s]",
+    fprintf(out, "%s/%s (IP: %u/%s)]",
```

Comment:
I would recommend to use square brackets to reduce confusion.

```suggestion
    fprintf(out, "%s/%s][IP: %u/%s]",
```

Reply:
Done